packages/compat/tests/resolver.test.ts

```diff
@@ -519,7 +519,7 @@ describe('compat-resolver', function () {
       },
     ]);
   });
-  test('string literal passed to `helper` helper in content position', function () {
+  test('string literal passed to "helper" keyword in content position', function () {
```

Comment:
I started renaming "helper" to "keyword" since that's the word they use in the error in ember-source:

> Uncaught Error: Assertion Failed: Passing a dynamic string to the `(helper)` keyword is disallowed. (You specified `(helper (concat "hel" "lo"))` and `(concat "hel" "lo")` evaluated into "hello".) This ensures we can statically analyze the template and determine which helpers are used. If the helper name is always the same, use a string literal instead, i.e. `(helper "hello")`. Otherwise, import the helpers into JavaScript and pass them to the helper keyword. See https://github.com/emberjs/rfcs/blob/master/text/0496-handlebars-strict-mode.md#4-no-dynamic-resolution for details. ('contextual-built-ins/templates/application.hbs' @ L39:C0)

Reply:
Yes, I agree, it seems clearer to say "keyword"